lib/token.h

```diff
@@ -460,12 +460,6 @@ class CPPCHECKLIB Token {
     void isSigned(const bool sign) {
         setFlag(fIsSigned, sign);
     }
-    bool isPointerCompare() const {
```

Comment:
We should keep this even if it isn't used. Same for the other methods in this file.